prime-router/src/main/kotlin/history/db/DeliveryDatabaseAccess.kt

```diff
@@ -167,76 +167,63 @@ class DeliveryDatabaseAccess(val db: DatabaseAccess = BaseEngine.databaseAccessS
-                        .concat(
-                            metadata.field(
-                                CovidResultMetadata
-                                    .COVID_RESULT_METADATA.ORDERING_FACILITY_NAME
-                            )
-                        )
-                        .concat(metadata.field(CovidResultMetadata.COVID_RESULT_METADATA.SENDER_ID))
-                        .`as`(DeliveryTable.DELIVERY.SORT_ID)
-                )
-                    .from(metadata)
-                    .join(ItemGraphTable.ITEM_GRAPH)
-                    .on(
-                        ItemGraphTable.ITEM_GRAPH.PARENT_REPORT_ID
-                            .eq(metadata.field(CovidResultMetadata.COVID_RESULT_METADATA.REPORT_ID)),
-                        ItemGraphTable.ITEM_GRAPH.PARENT_INDEX
-                            .eq(metadata.field(CovidResultMetadata.COVID_RESULT_METADATA.REPORT_INDEX))
                     )
-                    .join(ReportFile.REPORT_FILE)
-                    .on(ReportFile.REPORT_FILE.REPORT_ID.eq(ItemGraphTable.ITEM_GRAPH.STARTING_REPORT_ID))
-                    .groupBy(
-                        ReportFile.REPORT_FILE.REPORT_ID,
-                        metadata.field(
-                            CovidResultMetadata
-                                .COVID_RESULT_METADATA.ORDERING_PROVIDER_NAME
-                        ),
-                        metadata.field(CovidResultMetadata.COVID_RESULT_METADATA.ORDERING_FACILITY_NAME),
-                        metadata.field(CovidResultMetadata.COVID_RESULT_METADATA.SENDER_ID),
-                    ).coerce(DeliveryTable.DELIVERY)
+                    .concat(CovidResultMetadata.COVID_RESULT_METADATA.SENDER_ID)
+                    .`as`(DeliveryTable.DELIVERY.SORT_ID)
             )
+                .from(CovidResultMetadata.COVID_RESULT_METADATA)
+                .join(ItemGraphTable.ITEM_GRAPH)
+                .on(
+                    ItemGraphTable.ITEM_GRAPH.PARENT_REPORT_ID
+                        .eq(CovidResultMetadata.COVID_RESULT_METADATA.REPORT_ID),
+                    ItemGraphTable.ITEM_GRAPH.PARENT_INDEX
+                        .eq(CovidResultMetadata.COVID_RESULT_METADATA.REPORT_INDEX)
+                )
+                .join(ReportFile.REPORT_FILE)
+                .on(ReportFile.REPORT_FILE.REPORT_ID.eq(ItemGraphTable.ITEM_GRAPH.STARTING_REPORT_ID))
+                .groupBy(
+                    ReportFile.REPORT_FILE.REPORT_ID,
+
+                    CovidResultMetadata
+                        .COVID_RESULT_METADATA.ORDERING_PROVIDER_NAME,
+                    CovidResultMetadata.COVID_RESULT_METADATA.ORDERING_FACILITY_NAME,
+                    CovidResultMetadata.COVID_RESULT_METADATA.SENDER_ID,
+                ).asTable(DeliveryTable.DELIVERY)
+
             search.fetchResults(
                 DSL.using(txn),
                 DSL
                     .withRecursive(itemGraph)
-                    .with(metadata)
```

Comment:
This is something that we could put back later when the postgres update is complete, but Postgres 11 treats CTEs as optimization fences meaning it won't attempt to optimize that part of the query within the scope of the entire query, starting in postgres 12, if the CTE is only used once the optimizer will [inline](https://www.postgresql.org/docs/release/12.0/#:~:text=much%20better%20estimates.-,Allow,-common%20table%20expressions) it removing the optimization fence